pkg/local/utils.go

```diff
@@ -0,0 +1,251 @@
+package local
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	dockerclient "github.com/docker/docker/client"
+	"github.com/filecoin-project/bacalhau/pkg/config"
+	"github.com/filecoin-project/bacalhau/pkg/executor"
+	executor_util "github.com/filecoin-project/bacalhau/pkg/executor/util"
+	"github.com/filecoin-project/bacalhau/pkg/system"
+	"github.com/filecoin-project/bacalhau/pkg/transport/libp2p"
+	"github.com/moby/moby/pkg/stdcopy"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/rs/zerolog/log"
+)
+
+// ErrContainerMarkedForRemoval indicates that the docker daemon is about to
+// delete, or has already deleted, the given container.
+var ErrContainerMarkedForRemoval = fmt.Errorf("docker container marked for removal")
+
+var DefaultBootstrapAddresses = []string{
```

Comment:
Shouldn't this sit in some `config` file? Tho I see there's a a duplicate too in `bacalhau/cmd/bacalhau/serve.go`

Reply:
There are lot duplicate config values, Its a mess, I talked with kai about this
we need a single config file to avoid duplication we use enviroment variables, flags etc for configuration